packages/odata-service-writer/templates/extend/manifest.json

```diff
@@ -6,7 +6,8 @@
                 "type": "OData",
                 "settings": {
                     "annotations": [<%if (locals.annotations && annotations.technicalName) {%>
-                       "<%- annotations.technicalName %>"<% } %>
+                        "<%- annotations.technicalName %>"<% if (locals.localAnnotationsName) {%>,<%}%><% } %><% if (locals.localAnnotationsName) { %>
+                        "<%- localAnnotationsName %>"<% } %>
                     ],
                     "localUri": "localService/metadata.xml"<%if (version === '4') {%>,
                     "odataVersion": "4.0"<% } %><%if (version === '2') {%>,
```

Comment:
For completeness shouldn't we add odataVersion for V2 also. Makes it easier for other tools to check when working on the project.

Reply:
We do here https://github.com/SAP/open-ux-tools/pull/298/files/3e6e0ce7283018ce3d19fd73d0bc3f7169f28155#diff-36edc1a05a6eaa8b903f997faf779f4e228cf7ccd8aab52c8c124e6c520bf648L13 ;)